.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: dart
-sudo: false
 
-# Run against both the dev and channel.
+# necessary to avoid chrome sandboxing issues
+sudo: required
+addons:
+  chrome: stable
+  
+# Run against both the dev and stable channel.
 dart:
   - stable
   - dev
 
 # Define test tasks to run.
 dart_task:
   - test: --platform vm
+  - test: --platform chrome
```

Comment:
I'm not sure if it's easy to do, but it might be worth testing with Firefox too..

I would love to add safari and edge, but I understand if those are hard to automate :)